### PR TITLE
Change visibility of useful methods to support custom checks (DAT-16671)

### PR DESCRIPTION
**Pull request by wwillard7800**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [x] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: liquibase-dist/src/main/assembly/assembly-bin-common.xml
Comment (filipelautert) [inline liquibase-dist/src/main/assembly/assembly-bin-common.xml:1]: Did this file just have indentation fixes ?

Comment (wwillard7800) [reply liquibase-dist/src/main/assembly/assembly-bin-common.xml:1]: I wanted to get the file formatted before I made my real change.  There should be another commit after this one.

File: liquibase-dist/pom.xml [hunk 1]
```diff
@@ -27,11 +27,12 @@
 		<db2.version>11.5.9.0</db2.version>
 		<firebird.version>5.0.4.java8</firebird.version>
 		<liquibase-pro.version>master-SNAPSHOT</liquibase-pro.version>
+		<liquibase-scripting.version>0-SNAPSHOT</liquibase-scripting.version>
```
Comment (filipelautert) [inline liquibase-dist/pom.xml:30]: Should not be master-SNAPSHOT too?

File: liquibase-dist/pom.xml [hunk 2]
```diff
@@ -346,7 +347,27 @@
...
+                    <version>${liquibase-pro.version}</version>
+				</dependency>
+			</dependencies>
+		</profile>
+
+		<profile>
+			<id>liquibase-scripting</id>
+			<activation>
+				<property>
+            <name>liquibase.scripting</name>
```
Comment (filipelautert) [inline liquibase-dist/pom.xml:359]: Could you fix the indentation here?

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: liquibase-dist/pom.xml (hunk 1 above)
Comment (wwillard7800) [reply liquibase-dist/pom.xml:30]: This is confusing to me, too.  The artifact that is being produced right now is `0-SNAPSHOT`.  I think that once devops is finished with their build work it should be `master-SNAPSHOT`.

File: liquibase-dist/pom.xml (hunk 2 above)
Comment (wwillard7800) [reply liquibase-dist/pom.xml:359]: Yes.

**Review by filipelautert — CHANGES_REQUESTED** [review]
We need to review this one with Devops because of Bigquery work to make sure that both are implementing the same solution.

**Comment by filipelautert** [comment]
See DAT-18103

**Review by filipelautert — APPROVED** [review]

**Pull request merged**